guides/style-guide.md

```diff
@@ -1,3 +1,5 @@
+<!-- vale off -->
```

Comment:
I would argue that this document could be used as an example of how to turn off specific warnings well, in a way that doesn't turn them off for other bits of text. Does Vale support only disabling a specific rule?

Reply:
I think it's interesting and begs the question, what are these rules for? 

I don't think Vale is smart enough to really help out, like the `deliver` example you mention, it's really more a que to the user writing the content to check their use of `deliver` is the right one, see: 

https://github.com/dxw/playbook/blob/c905e043f11fcc18e11adbd85f26c3f24b6f7be1/styles/govuk/words-to-avoid.yml#L12

It looks like Vale has a `suggestion` setting for this, but Github won't surface the suggestions so it would not help a lot of the people who create the content for the Playbook. I assume none of these warnings have ever been seen by folk not running Vale locally?

In terms of disabling, it looks like you can change the level of the error Vale raises but not skip a specific rule, I might be wrong though:

https://errata-ai.github.io/vale/config/#ignoring-parts-of-your-documents

Reply:
This is what I was expecting:

```md
<!-- vale Style.Rule = NO -->

This is some text

<!-- vale Style.Rule = YES -->
```

which would disable `Style.Rule` for the text in the block.

> I assume none of these warnings have ever been seen by folk not running Vale locally?

I would think that's right, but they will _start_ to see them with this change, which means we need to be able to tell people how to fix them. I think having rules are useful (see Sensitivity Bot), but allowing people to override them, and say that they have read the warning, but think they are right, regardless, is a good thing. It also documents that someone has _decided_ that their choice of words is intentional.

Reply:
Agreed, I guess I am not comfortable to add an extra layer of complexity, no matter how small, to getting changes into the Playbook without consultation.

As these are only warnings, or suggestions now, they don't block merging the content into the playbook so a user can still choose to ignore them through that process?

I also cannot seem to get `<!-- vale Style.Rule = NO -->` to work, tired `<!-- vale Style.Rule = NO -->` and specific ones like `<!-- vale dxw.tired-idioms = NO -->` which is what I had assumed the syntax was.

Reply:
All I can think of is we don't follow the CamelCase rules explicitly for our styles, so it should be:

`<!-- vale Dxw.TiredIdioms = NO -->`

Reply:
Yep, looks like the camelcase is required and it's a bit sensitive to where the comments are. I'll have to look at renaming the styles and rules.